charts/cluster-registrator/templates/job.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: nirmata
+  name: {{ .Values.namespace }}
```

Comment:
Improvement: Let us remove the namespace creation in the job. So by default it will use the namespace of the registrator helm chart.